cms/templates/structured_tags_block.html

```diff
@@ -1,16 +1,16 @@
-<div class="xblock-render">
+<div class="xblock-render" class="studio-xblock-wrapper">
     % for tag in tags:
-    <label for="problem_tags_${tag['key']}">${tag['title']}</label>:
-    <select id="problem_tags_${tag['key']}" name="${tag['key']}">
+    <label for="tags_${tag['key']}">${tag['title']}</label>:
+    <select id="tags_${tag['key']}" name="${tag['key']}">
```

Comment:
Question: Will this behave correctly if there are multiple `structured_tags_blocks` on the same page? In particular, I'm a bit worried that the id isn't going to be unique in that case.